cmd/remove.go

```diff
@@ -45,5 +50,19 @@ func findRepo(repoName string) (entry, error) {
 		}
 	}
 
-	return entry{}, errors.New("Repository not found.")
+	return entry{}, errors.New("repository not found")
+}
+
+func removeRepo(repoName string) error {
+	repo, err := findRepo(repoName)
+	if err != nil {
+		return fmt.Errorf("repository `%s` not found", repoName)
```

Comment:
`fmt.Errorf` 👍 

Can we change the message to `"Repository '%s' not found."` (backticks around the `%s`)